src/components/IntegrationsSection.tsx

```diff
@@ -87,8 +87,10 @@ const renderListItems = (items: string[]): ReactNode =>
         </li>
     ))
 
-export const IntegrationsSection: FunctionComponent = () => (
-    <ContentSection className="py-7">
+export const IntegrationsSection: FunctionComponent<{
+    className?: string
+}> = ({ className = '' }) => (
```

Comment:
I'm confused why this `className` prop is added since I don't see it used. Is a class passed to this component somewhere?

Reply:
This is an error on my part from I was testing ways to add the light gray background. I ended up wrapping it instead, as we have done elsewhere, so this should be removed. d06d69e